visual-js/visual-cypress/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM cypress/included:13.11.0 AS runner
```

```suggestion
FROM cypress/included:latest AS runner
```
Should'nt we use `latest` to be sure we always test it against the latest version of Cypress ?
Otherwise I'm concerned we will forget to upgrade that value.